ycms/cms/forms/bed_assignment/bed_assignment_form.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from django import forms
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from ...models import Bed, BedAssignment, MedicalRecord
+from ..custom_model_form import CustomModelForm
+
+logger = logging.getLogger(__name__)
+
+
+class BedAssignmentForm(CustomModelForm):
+    """
+    Form for creating bed assignments.
+    """
+
+    class Meta:
+        """
+        This class contains additional meta configuration of the form class, see the :class:`django.forms.ModelForm`
+        for more information.
+        """
+
+        model = BedAssignment
+        fields = ["medical_record", "admission_date", "discharge_date"]
+        widgets = {
+            "admission_date": forms.NumberInput(
```

Comment:
```suggestion
        widgets = {
            "medical_record": forms.HiddenInput(),
            "admission_date": forms.NumberInput(
```
This way the field will not be shown to the user at all.